deployment/agent-deployment/ServiceGroupRoot/Scripts/pushAgentToAcr.sh

```diff
@@ -38,79 +38,22 @@ if [ -z $ACR_NAME ]; then
   exit 1
 fi
 
-#Download agentimage tarball from blob storage account
-echo "Downloading tarball image from $WINDOWS $AGENT_IMAGE_URI"
-wget -O $AGENT_IMAGE_TAR_FILE_NAME "${AGENT_IMAGE_URI}${WINDOWS}${RELEASE_ID}${AGENT_IMAGE_SAS}"
-
-
-if [ ! -f $AGENT_IMAGE_TAR_FILE_NAME ]; then
-    echo "Agent tarfile: ${AGENT_IMAGE_TAR_FILE_NAME} does not exist, unable to continue"
-    exit 1
-fi
-
-#Install crane
-echo "Installing crane"
-wget -O crane.tar.gz https://github.com/google/go-containerregistry/releases/download/v0.4.0/go-containerregistry_Linux_x86_64.tar.gz
-if [ $? -eq 0 ]; then         
-   echo "crane downloaded successfully"
-else     
-   echo "-e error crane download failed"
-   exit 1
-fi 
-tar xzvf crane.tar.gz
-echo "Installed crane"
-
 
 #Login to az cli and authenticate to acr
 echo "Login cli using managed identity"
 az login --identity
-
-echo "Getting acr credentials"
-TOKEN_QUERY_RES=$(az acr login -n "$ACR_NAME" -t)
-TOKEN=$(echo "$TOKEN_QUERY_RES" | jq -r '.accessToken')
-if [ -z $TOKEN ]; then
-  echo "-e error failed to get az acr login token"
+if [ $? -eq 0 ]; then
+  echo "Logged in successfully"
+else
+  echo "-e error failed to login to az with managed identity credentials"
   exit 1
-fi
+fi     
 
-DESTINATION_ACR=$(echo "$TOKEN_QUERY_RES" | jq -r '.loginServer')
-if [ -z $DESTINATION_ACR ]; then
-  echo "-e error value of DESTINATION_ACR shouldnt be empty"
+echo "Pushing ${AGENT_IMAGE_FULL_PATH} to ${ACR_NAME}"
+az acr import --name $ACR_NAME --registry $CDPX_REGISTRY --source $CDPX_ACR:$CDPX_TAG --image $AGENT_IMAGE_FULL_PATH
```

Comment:
please note, full of the CDPx ACR in this format - https://github.com/microsoft/Docker-Provider/blob/ci_prod/.pipelines/pull-from-cdpx-and-push-to-ci-acr-linux-image.sh#L63
https://github.com/microsoft/Docker-Provider/blob/ci_prod/.pipelines/pull-from-cdpx-and-push-to-ci-acr-windows-image.sh#L61